AlphaWallet/Core/Initializers/WalletMigrationInitializer.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  WalletMigrationInitializer.swift
+//  AlphaWallet
+//
+//  Created by Vladyslav Shepitko on 09.05.2022.
+//
+
+import Foundation
+
+class WalletMigrationInitializer: Initializer {
```

Comment:
Would you help rename `WalletMigrationInitializer` to something like `CleanupWallets`?

1. clearer, and
2. we should drop the "Initializer" suffix since the name makes (better) sense without it